### DISCUSSION TITLE
在 iPlay40 Pro 上安装 LineageOS + Magisk

### DISCUSSION THREAD

**Discussion by giscus** [kxxt/kxxt-website#167] [Thread]
# 在 iPlay40 Pro 上安装 LineageOS + Magisk

在踩了很多坑之后，我终于在 iPlay40 Pro 上用上了 Lineage OS + Magisk. 于是我写下了这篇博客作为记录，同时也希望会对别人有所帮助。

https://www.kxxt.dev/blog/lineageos-and-magisk-on-iplay40pro/

**Comment by pehuang** [comment]
請問您有iPlay40  Pro最新國際板韌體的magisk_match.img以及vbmeta-sign-custom.img嗎? 目前您分享的這兩份img在最新韌體下(iPlay40H(T1020H)-EN-20221027)刷機後電池容量會卡在0%。
